README.md

```diff
@@ -163,13 +164,13 @@ This SDK provides a basic serialization for request/response objects but also fo
     ```
 
 
-### Debug Java application or Dapr's Java SDK
+### Debug a Java application or Dapr's Java SDK
 
-**In IntelliJ Community Edition, consider [debugging in IntelliJ](https://docs.dapr.io/developing-applications/ides/intellij/).**
+**In IntelliJ Community Edition, consider [debugging in IntelliJ](https://www.jetbrains.com/help/idea/debugging-your-first-java-application.html).**
```

Comment:
Is there a reason to change this link?

Reply:
that original link is a 404: Not Found

Reply:
The original page appears to have moved to https://docs.dapr.io/developing-applications/local-development/ides/intellij/

It would be useful to include this link rather than the generic Jetbrains page, as there is specific information for setting up a run/debug configuration for DAPR.